nixui/graphics/diff_widget.py

```diff
@@ -51,15 +58,31 @@ class DiffDialog(QtWidgets.QDialog):
     def __init__(self, statemodel, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        diff_table = DiffedOptionListSelector(statemodel.get_update_set())
+        self.statemodel = statemodel
 
-        btn_box = QtWidgets.QDialogButtonBox(QtWidgets.QDialogButtonBox.Ok)
-        btn_box.accepted.connect(self.accept)
+        diff_table = DiffedOptionListSelector(statemodel.get_update_set())
 
         layout = QtWidgets.QVBoxLayout()
         layout.addWidget(diff_table)
-        layout.addWidget(btn_box)
+        layout.addWidget(self.get_btn_box())
 
         self.setSizePolicy(QtWidgets.QSizePolicy.Maximum, QtWidgets.QSizePolicy.Maximum)
 
         self.setLayout(layout)
+
+    def get_btn_box(self):
```

Comment:
I wouldn't call this `get_btn_box` maybe `init_btn_box`? Or something similar.

Reply:
done